ludwig/schema/parameter_metadata.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, List, Union
+
+
+class ExpectedImpact(Enum):
+    """The expected impact of determining a "good" value for a specific parameter.
+
+    - HIGH: this parameter should almost always be included in a hyperopt run and can make or break a good model.
+    - MEDIUM: this parameter can sometimes make or break a good model.
+    - LOW: this parameter usually does not have a significant impact on model performance.
+    """
+
+    UNKNOWN = 0
```

Comment:
What is the rationale for having an `UNKNOWN` flag? I would assume default as `LOW`

Reply:
`UNKNOWN` allows us to distinguish parameters that haven't been annotated vs. parameters that have been explicitly marked as having `LOW` impact.